api/nodes/serializers.py

```diff
@@ -53,6 +53,16 @@ def to_internal_value(self, data):
         provider = self.get_object(data)
         return {'provider': provider}
 
+
+class CollectionProviderRelationshipField(RelationshipField):
```

Comment:
Why is this living in the node serializers?

Reply:
Moved to `api/collections_providers/fields.py`